katas/langtonant/maria/prueba.html

```diff
@@ -0,0 +1,8 @@
+<!DOCTYPE html>
```

Comment:
This file should not be needed, can be removed?

Reply:
Yes, it has been removed in the last commit.